admin/spec/components/previews/solidus_admin/sidebar/item/component_preview.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "solidus_admin/main_nav_item"
+
+# @component "sidebar/item"
+class SolidusAdmin::Sidebar::Item::ComponentPreview < ViewComponent::Preview
+  include SolidusAdmin::Preview
+
+  DUMMY_ROUTE = :foo_path
+
+  DUMMY_PATH = "#"
+
```

Comment:
What about using the notes to document a bit the `url_helpers` and `request` parameters, as we did [here](https://github.com/solidusio/solidus/blob/a9bba258ef3ca2ac72d854e60927ddaff1a6dd89/admin/spec/components/previews/solidus_admin/ui/forms/checkbox/component_preview.rb#L7-L47)?

Reply:
I feel a bit uneasy documenting those parameters, which are private in nature, in the lookbook documentation. In any case, I added docs about how to build the component from a main navigation item, and also documented all the parameters (including `url_helpers` & `request`) in the component class. What do you think?